v4/source/posts.yaml

```diff
@@ -417,6 +417,48 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/users/{user_id}/channels/{channel_id}/posts/unread':
+    get:
+      tags:
+        - posts
+      summary: Get posts around last unread
+      description: |
+        Get the oldest unread post in the channel for the given user as well as the posts around it.
+        ##### Permissions
+        Must be logged in as the user or have `edit_other_users` permission, and must have `read_channel` permission for the channel.
```

Comment:
Actually can we add the min server version?

Reply:
Thanks Joram, I forgot to push my changes last week.  Should be good now, please take a look again.